src/rules/guard-define-call.ts

```diff
@@ -27,7 +27,7 @@ const rule: Rule.RuleModule = {
 
   create(context): Rule.RuleListener {
     const definedCustomElements = new Set<string>();
-
+    const sourceCode = context.sourceCode ?? context.getSourceCode();
```

Comment:
`context.sourceCode` was introduced in 9.x iirc, so we may not need to do this

pretty sure we have other places where we depend on `>=9` too